src/views/homepage/components/chiclets/chiclets.module.css

```diff
@@ -0,0 +1,41 @@
+.root {
+	margin-top: 72px;
+}
+
+.label {
+	composes: hds-typography-display-100 from global;
+	color: var(--token-color-foreground-high-contrast);
+	font-weight: var(--token-typography-font-weight-medium);
+}
+
+.nav {
+	margin-top: 30px;
+}
+
+.navList {
+	display: flex;
+	flex-wrap: wrap;
+	gap: 20px;
+	list-style: none;
+	margin: 0;
+	padding: 0;
+	width: 100%;
+}
+
+.navListLink {
+	composes: g-focus-ring-from-box-shadow from global;
+	background-color: var(--token-color-foreground-strong);
+	border-radius: 10px;
+	box-shadow: 0 0 0 1px rgba(255, 255, 255, 0.1);
```

Comment:
Is there a token value we can use for this? One of the alpha tokens?

Reply:
No, there's no token that I could find!

Reply:
We should ask design if they are sure they want to use a bespoke value here. I'd prefer to push back for `alpha-100` or something. Or perhaps we define a new top-level alpha value for this if they are going to reuse it elsewhere?

Reply:
Chatting w/ Kelly now -- we'll update this to `--token-color-border-strong`, which has some opacity built in!

Reply:
Oh perfect!

Reply:
Updated in https://github.com/hashicorp/dev-portal/pull/1763/commits/bb297c944f49adc4eac6316f72dc4e7da33e5ac0! ✅